src/main/webapp/app/core/about-us/about-us.component.html

```diff
@@ -56,4 +56,8 @@ <h3 class="font-weight-bold">{{ 'artemisApp.aboutUsOverview.contributors' | arte
             </div>
         </div>
     </div>
+    <div class="mt-5 ms-2 me-2">
+        <h3 class="font-weight-bold">Git Information</h3>
+        <div>Git Branch: {{ gitBranchName }}<br />Git Commit ID: {{ gitCommitId }}</div>
```

Comment:
Even though the translations might be identical here, please move all text to the translation files.

Reply:
Done :)